Detectors/FIT/FT0/simulation/src/Detector.cxx

```diff
@@ -358,8 +358,6 @@ Bool_t Detector::ProcessHits(FairVolume* v)
 o2::ft0::HitType* Detector::AddHit(float x, float y, float z, float time, float energy, Int_t trackId, Int_t detId)
 {
   mHits->emplace_back(x, y, z, time, energy, trackId, detId);
-  auto stack = (o2::data::Stack*)fMC->GetStack();
-  stack->addHit(GetDetId());
```

Comment:
this means you not adding **any** hit leaving particle to the stack. Is that ok for you?

Reply:
No, it was just panic solution

Reply:
There is no need to panic... For the moment this is not a critical issue and we can work out a proper solution.